### PR TITLE
feat(input): add getInput method and document how to use maskito

### PR DESCRIPTION
**Pull request by micahjones13**
## Brief Description
Adds a new method to rux-input called getInput. This method will return the shadow-dom'd input that we use. 
Adds documentation to storybook on how to use the Maskito library to create custom masks of rux-input, with the example being a Julian input. 
## JIRA Link

https://rocketcom.atlassian.net/browse/ASTRO-6595

## Motivation and Context

Julian datepicker/calendar needed a julian formatted input 

## Types of changes

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## Checklist

- [ ] This PR adds or removes a Storybook story.
- [ ] I have added tests to cover my changes.
- [ ] Regressions are passing and/or failures are documented
- [ ] Changes have been checked in evergreen browsers

### CODE REVIEW

File: packages/web-components/src/utils/utils.ts [hunk 1]
```diff
@@ -1,3 +1,14 @@
+import { Maskito } from '@maskito/core'
+
+export function mask(inputElement: HTMLInputElement) {
+    console.log(inputElement, 'input el')
```
Comment (FMorrison87) [inline packages/web-components/src/utils/utils.ts:4]: Remove console log?

**Review by FMorrison87 — APPROVED** [review]
Left a few minor comments. Pop in a changeset and you'll be good to go. Nice work!

File: packages/web-components/src/utils/utils.ts [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+    if (inputElement) {
+        new Maskito(inputElement, {
+            // 1234/123
+            mask: [/\d/, /\d/, /\d/, /\d/, '/', /\d/, /\d/, /\d/],
+        })
+    } else return //? wtf should I return if el isn't defined
```
[the remark this answers is not in the document]
Comment (kiley-mitti) [reply packages/web-components/src/utils/utils.ts:10]: Probably don't need the else :)

**Review by kiley-mitti — APPROVED** [review]

**Review by kiley-mitti — APPROVED** [review]
One tiny change and it needs a changeset and all good!

**Review by markacianfrani — APPROVED** [review]

**Pull request merged**